src/Passes/libs/ExpandStaticAllocation/SPECIFICATION.md

```diff
@@ -0,0 +1 @@
+# {ExpandStaticAllocation} Specification
```

Comment:
I've added a `SPECIFICATION.md` to each of the passes. The initial thought was that each of the pass should have a specification such that it is clear what it does without the need to read the code. However, this may be revised in one of the future PRs.